src/main/java/anya/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package anya.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents a generic task in the Anya application.
+ * A task has a description and a completion status.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructs a new `Task` instance with the specified description and initializes it as not done.
+     *
+     * @param description A description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
```

Comment:
Shouldn't there be a JavaDoc comment for this method?  I noticed the same issue in several other places too.